### PR TITLE
Allow filtering triggers by worker type

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: web/jobs/jobs.go [hunk 1]
```diff
@@ -226,6 +226,7 @@ func deleteTrigger(c echo.Context) error {
 
 func getAllTriggers(c echo.Context) error {
 	instance := middlewares.GetInstance(c)
+	workerFilter := c.QueryParam("worker")
```
Comment (jinroh) [inline web/jobs/jobs.go]: To be consistent, we should use a capitalized query param `Worker`.

File: docs/konnectors_workflow_example.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ See https://cozy.github.io/cozy-stack/jobs.html#post-jobstriggers
 - [ ] `GET    /konnectors`             Lists installed konnectors    
 
 **triggers**
-- [ ] `GET    /jobs/triggers?worker=konnector` Lists konnectors with a configured recurrence.
+- [ ] `GET    /jobs/triggers?Worker=konnector` Lists konnectors with a configured recurrence.
```
Comment (nono) [inline docs/konnectors_workflow_example.md]: `s/[ ]/[x]/` ?

File: web/jobs/jobs_test.go [hunk 3]
```diff
@@ -401,6 +401,27 @@ func TestGetAllJobs(t *testing.T) {
 		assert.Equal(t, "10s", v.Data[index].Attributes.Arguments)
 		assert.Equal(t, "print", v.Data[index].Attributes.WorkerType)
 	}
+
+	req4, err := http.NewRequest(http.MethodGet, ts.URL+"/jobs/triggers?worker=print", nil)
```
Comment (nono) [inline web/jobs/jobs_test.go]: s/worker/Worker/

Comment (nono) [inline web/jobs/jobs_test.go]: Well, travis is still green. Can you add a test to check that if you ask for another `Worker`, you have 0 results?

**Pull request merged**